scrapi/processing/elastic_search.py

```diff
@@ -0,0 +1,57 @@
+import logging
+
+from elasticsearch import Elasticsearch
+from elasticsearch.exceptions import NotFoundError
+
+from scrapi import settings
+from scrapi.processing.base import BaseProcessor
+
+es = Elasticsearch(
+    settings.ELASTIC_URI,
+    request_timeout=settings.ELASTIC_TIMEOUT
+)
+
+logging.getLogger('elasticsearch').setLevel(logging.WARN)
+logging.getLogger('elasticsearch.trace').setLevel(logging.WARN)
+logging.getLogger('urllib3').setLevel(logging.WARN)
+logging.getLogger('requests').setLevel(logging.WARN)
+es.cluster.health(wait_for_status='yellow')
+
+logger = logging.getLogger(__name__)
+
+
+class ElasticsearchProcessor(BaseProcessor):
+    NAME = 'elasticsearch'
+
+    def process_normalized(self, raw_doc, normalized):
+        data = {
+            key: value for key, value in normalized.attributes.items()
+            if key in settings.FRONTEND_KEYS
```

Comment:
1. No, each document is pretty much unique (ideally)
2. At the moment you can't specify a nested field (you can include it by specifying the top level field)
3. descriptive?